<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeView</class>
 <widget class="QWidget" name="WelcomeView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>320</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>320</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>800</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>What should we work on?</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(33, 33, 33);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>500</width>
     <height>320</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>500</width>
     <height>320</height>
    </size>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="QFrame" name="frame">
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <property name="lineWidth">
       <number>0</number>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="spacing">
        <number>0</number>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QFrame" name="left">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QFrame" name="darkblue">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>0</y>
            <width>31</width>
            <height>321</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(38, 133, 161);</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="lightblue">
          <property name="geometry">
           <rect>
            <x>60</x>
            <y>0</y>
            <width>21</width>
            <height>321</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(105, 166, 184);</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QFrame" name="pink">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>0</y>
            <width>31</width>
            <height>321</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(255, 146, 169);</string>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
         </widget>
         <widget class="QLabel" name="girl_with_cat">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>50</y>
            <width>110</width>
            <height>320</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">QLabel{ background-color: transparent;}</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="pixmap">
           <pixmap resource="../resources/resources.qrc">:/images/images/girl-with-cat.png</pixmap>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QFrame" name="right">
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLabel" name="big_label">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>80</y>
            <width>281</width>
            <height>51</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <pointsize>18</pointsize>
            <bold>true</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(221, 221, 221);</string>
          </property>
          <property name="text">
           <string>What should we work on?</string>
          </property>
         </widget>
         <widget class="QLabel" name="description">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>120</y>
            <width>281</width>
            <height>71</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true">color: rgb(216, 216, 216);</string>
          </property>
          <property name="text">
           <string>Show us the folders whose contents you want us to group for you. Don't worry, we won't do anything to your files.</string>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QPushButton" name="button_select_folder">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>200</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
  padding: 0;
  overflow: hidden;
  border-width: 0;
  outline: none;
  border-radius: 2px;
  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
  background-color: #bb86fc;
  color: black;
}</string>
          </property>
          <property name="text">
           <string>Load from folder</string>
          </property>
         </widget>
         <widget class="QPushButton" name="button_load_saved_data">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>10</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">QPushButton
{
  padding: 0;
  overflow: hidden;
  border-width: 0;
  outline: none;
  border-radius: 2px;
  box-shadow: 0 1px 4px rgba(0, 0, 0, .6);
  background-color: #bb86fc;
  color: black;
}</string>
          </property>
          <property name="text">
           <string>Load from file</string>
          </property>
         </widget>
         <widget class="QPushButton" name="button_threedots">
          <property name="geometry">
           <rect>
            <x>280</x>
            <y>280</y>
            <width>25</width>
            <height>10</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>10</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>25</width>
            <height>10</height>
           </size>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">border: 0px soliid;</string>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="icon">
           <iconset resource="../resources/resources.qrc">
            <normaloff>:/icons/icons/three-dots.png</normaloff>:/icons/icons/three-dots.png</iconset>
          </property>
          <property name="iconSize">
           <size>
            <width>30</width>
            <height>30</height>
           </size>
          </property>
         </widget>
         <widget class="QPushButton" name="button_select_algorithm">
          <property name="geometry">
           <rect>
            <x>170</x>
            <y>240</y>
            <width>150</width>
            <height>30</height>
           </rect>
          </property>
          <property name="minimumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>150</width>
            <height>30</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Segoe UI</family>
            <pointsize>9</pointsize>
            <bold>false</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="styleSheet">
           <string notr="true">background-color: rgb(47, 47, 47);
border: 0px;
box-shadow: 8px 8px 24px 0px rgba(0, 0, 0, 1);
color: rgb(177, 177, 177);</string>
          </property>
          <property name="text">
           <string>Change way of grouping</string>
          </property>
         </widget>
         <widget class="QLabel" name="unavailable">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>240</y>
            <width>131</width>
            <height>31</height>
           </rect>
          </property>
          <property name="styleSheet">
           <string notr="true">color: white;
background-color: rgba(0, 0, 0, 180);</string>
          </property>
          <property name="text">
           <string>Temporarily unavailable</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
